hack/move-generated-helm-charts.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
Reorganize files that kustomize generated, so they are similar to what we have currently in the repo (easier to review)